Dockerfile.builder

```diff
@@ -5,32 +5,34 @@ ENV LANG=en_US.utf8
 ARG USE_GO_VERSION_FROM_WEBSITE=0
 
 # Some packages might seem weird but they are required by the RVM installer.
-RUN yum --enablerepo=centosplus install -y --quiet \
+RUN yum install epel-release -y \
+    && yum --enablerepo=centosplus --enablerepo=epel install -y \
       findutils \
       git \
-      $(test -z $USE_GO_VERSION_FROM_WEBSITE && echo "golang") \
+      $(test "$USE_GO_VERSION_FROM_WEBSITE" != 1 && echo "golang") \
       make \
       procps-ng \
       tar \
       wget \
       which \
+      bc \
```

Comment:
It's not a blocker, just a question - why is `bc` needed?

Reply:
It's not needed. Just copy-paste ;)
I've removed it.